content/en/docs/contribute/localization.md

```diff
@@ -328,74 +340,92 @@ mkdir -p content/de/docs/tutorials
 cp content/en/docs/tutorials/kubernetes-basics.md content/de/docs/tutorials/kubernetes-basics.md
 ```
 
-Translation tools can speed up the translation process. For example, some editors offers plugins
-to quickly translate text.
+Translation tools can speed up the translation process. For example, some
+editors offer plugins to quickly translate text.
 
 {{< caution >}}
-Machine-generated translation is insufficient on its own. Localization requires extensive human
-review to meet minimum standards of quality.
+Machine-generated translation is insufficient on its own. Localization requires
+extensive human review to meet minimum standards of quality.
 {{< /caution >}}
 
-To ensure accuracy in grammar and meaning, members of your localization team should carefully
-review all machine-generated translations before publishing.
+To ensure accuracy in grammar and meaning, members of your localization team
+should carefully review all machine-generated translations before publishing.
 
 ### Source files
 
-Localizations must be based on the English files from a specific release targeted by the localization team.
-Each localization team can decide which release to target which is referred to as the _target version_ below.
+Localizations must be based on the English files from a specific release
+targeted by the localization team. Each localization team can decide which
+release to target, referred to as the _target version_ below.
 
 To find source files for your target version:
 
-1. Navigate to the Kubernetes website repository at https://github.com/kubernetes/website.
-2. Select a branch for your target version from the following table:
+1. Navigate to the Kubernetes website repository at
+https://github.com/kubernetes/website. 2. Select a branch for your target
+version from the following table:
 
    Target version | Branch
    -----|-----
    Latest version | [`main`](https://github.com/kubernetes/website/tree/main)
    Previous version | [`release-{{< skew prevMinorVersion >}}`](https://github.com/kubernetes/website/tree/release-{{< skew prevMinorVersion >}})
    Next version | [`dev-{{< skew nextMinorVersion >}}`](https://github.com/kubernetes/website/tree/dev-{{< skew nextMinorVersion >}})
 
-The `main` branch holds content for the current release `{{< latest-version >}}`. The release team
-will create a `{{< release-branch >}}` branch before the next release: v{{< skew nextMinorVersion >}}.
+The `main` branch holds content for the current release `{{< latest-version >}}`. The release team creates a `{{< release-branch >}}` branch before the next
```

Comment:
Please wrap long lines like this appropriately. Long lines are difficult for diff tracking.